configs/dev-emulator-program.nrel-op.json

```
@@ -3,6 +3,9 @@
     "ts": 1655143472,
     "intro": {
         "program_or_study": "program",
+        "start_month": "9",
+        "start_year": "2020",
+        "mode_studied": "pilot_ebike",
```

```suggestion
        "mode_studied": "e-bike",
```

Since OpenPATH is designed for all kinds of studies, we no longer call it the `pilot_ebike`. Instead, it is `e-bike`.
https://github.com/e-mission/e-mission-phone/blob/master/www/json/trip_confirm_options.json.sample#L5